cico_build_deploy.sh

```diff
@@ -52,10 +52,17 @@ rm -rf ${TARGET_DIR}/
 docker build -t ${BUILDER_IMAGE} -f Dockerfile.build .
 
 mkdir ${TARGET_DIR}/
+mkdir ${TARGET_DIR}/images
+
+
 docker run --detach=true --name ${BUILDER_CONT} -t -v $(pwd)/${TARGET_DIR}:/${TARGET_DIR}:Z ${BUILDER_IMAGE} /bin/tail -f /dev/null #FIXME
 
+#This sets the permissions on target dir since this script may be invoked as root, but container is not.
+chmod -R 0777 ${TARGET_DIR}/
```

Comment:
This needs to happen _after_ `docker exec ${BUILDER_CONT} sh scripts/build_guides.sh` because that script creates files in the `html` directory as root, so even if you run `chmod` now, the regular user won't be able to delete them later.